src/test/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/location/LocationTypeTest.kt

```diff
@@ -0,0 +1,105 @@
+package uk.gov.justice.digital.hmpps.pecs.jpc.location
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+
+class LocationTypeTest {
+
+    @Test
+    fun `airport location type is mapped`() {
```

Comment:
Lots of duplication in these tests - possibly a case for `@ParameterizedTest`?

Reply:
Thanks. That's a fair point.

Reply:
Done.